src/utils.ts

```diff
@@ -3,6 +3,8 @@ import chalk from "chalk";
 export const getEtherscanAddress = ({ network }: { network: string }): string =>
   `https://${network === "mainnet" ? "" : `${network}.`}etherscan.io`;
 
+export const validateAddressPrefix = (address: string): string => (address.startsWith("0x") ? address : `0x${address}`);
+
```

Comment:
it's not really called validating, more of adding `0x` prefix. https://ethereum.stackexchange.com/questions/503/what-is-the-0x-i-see-around-ethereum-for-example-i-see-addresses-that-start-wit

Reply:
ok, changed it to addAddressPrefix()